qiskit/providers/ibmq/api/clients/websocket.py

```diff
@@ -261,11 +290,11 @@ def get_job_status(
                                 timeout = original_timeout - (time.time() - start_time)
                             else:
                                 response_raw = yield from websocket.recv()
-                        logger.debug('Received message from websocket: %s',
-                                     response_raw)
 
                         response = WebsocketResponseMethod.from_bytes(response_raw)
                         last_status = response.data
+                        logger.debug('Received message from websocket: %s',
+                                     response.get_data_filtered())
```

Comment:
`get_data_filtered()` does a deepcopy(), which is not really an issue since the data is tiny. However, you might still want to check the log level before calling it in case data gets bigger in the future.